src/dataregistry/registrar.py

```diff
@@ -19,6 +19,9 @@
 # Default maximum allowed length of configuration file allowed to be ingested
 _DEFAULT_MAX_CONFIG = 10000
 
+# Allowed owner types
+_OWNER_TYPES = {"user", "project", "group", "production"}
```

Comment:
Slightly prefer
OWNER_TYPES = ...
that is, omit the initial underscore.   Alternatively, make a little routine which returns the value.  One way or another, that value should be accessible to users.